internal/go/skycfg/assert.go

```diff
@@ -113,9 +113,10 @@ func (t *TestContext) CallInternal(thread *starlark.Thread, args starlark.Tuple,
 
 	if !val {
 		buf := new(strings.Builder)
-		thread.Caller().WriteBacktrace(buf)
+		stk := thread.CallFrame(1)
+		fmt.Fprintf(buf, "%s", stk)
```

Comment:
I'm confused about the content of `buf` at this point. The docs and definition for `CallFrame` imply it's a single frame from the stack, so stringifying it won't provide a full stack trace.